src/pydocstyle/cli.py

```diff
@@ -53,7 +53,7 @@ def run_pydocstyle(use_pep257=False):
 
     count = 0
     for error in errors:
-        sys.stderr.write('%s\n' % error)
+        sys.stdout.write('%s\n' % error)
```

Comment:
Now that errors are prints to `stdout`, notice that the test for `--count` in `test_integration.py` needs to be made more robust.